hedera-state-validator/README.md

```diff
@@ -61,17 +61,14 @@ of a corrupted state.
 2. Run the following command to execute the export:
 
    ```shell
-   java -jar [-DmaxObjPerFile=X] [-Dsorted=true] [-DprettyPrint=true] ./validator-<version>.jar {path-to-state-round} export {path-to-result-dir} [{service_name}] [{state_key}]
+   java -jar [-DmaxObjPerFile=X] [-DprettyPrint=true] ./validator-<version>.jar {path-to-state-round} export {path-to-result-dir} [{service_name}] [{state_key}]
```

Comment:
I assume this system property, `-Dsorted.true`, is used in some github workflows, maybe in a different repo. They should be updated, too

Reply:
I opened PR in for updating github workflows